wallets/rn_cli_wallet/patches/react-native-minimizer+1.3.5.patch

```diff
@@ -0,0 +1,110 @@
+diff --git a/node_modules/react-native-minimizer/android/build.gradle b/node_modules/react-native-minimizer/android/build.gradle
```

Comment:
What is react-native-minimizer package for and could we possibly not use it to avoid patching?

Reply:
its a lib that provides a method to go back to the previous opened app. I have to patch it to make it work with the latest version of RN. The other option is to remove that logic, and simply show a toast in case deeplinking doesn't work.

This lib is used by MM and Rainbow too

Reply:
@0xAsimetriq thoughts?

Reply:
Sounds good 👍